package.json

```diff
@@ -3,12 +3,40 @@
   "productName": "Sabaki",
   "version": "0.20.0",
   "description": "An elegant Go/Baduk/Weiqi board and SGF editor for a more civilized age.",
-  "author": "Yichuan Shen",
+  "author": "Yichuan Shen <shen.yichuan@gmail.com>",
+  "homepage": "http://sabaki.yichuanshen.de",
+  "license": "MIT",
   "main": "main.js",
   "repository": {
     "type": "git",
     "url": "https://github.com/yishn/Sabaki"
   },
+  "build": {
+    "appId": "de.yichuanshen.sabaki",
+    "copyright": "Copyright © 2015-2017 Yichuan Shen",
+    "linux": {
+      "category": "Game",
+      "packageCategory": "games",
+      "target": ["deb", "rpm", "pacman", "tar.gz"]
+    },
+    "mac": {
+      "category": "public.app-category.board-games"
+    },
+    "win": {
+      "iconUrl": "https://raw.githubusercontent.com/yishn/sabaki/master/data/icon.ico",
+      "target": ["msi"]
```

Comment:
Is there a `msi` target for Windows? It doesn't work on my machine and I couldn't find the target in the [documentation](https://github.com/electron-userland/electron-builder/wiki/Options#WinBuildOptions-target). Maybe we can just provide a 7z or use NSIS?